src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/FallbackProjectManager.cs

```diff
@@ -98,16 +112,16 @@ private void AddFallbackProject(ProjectId projectId, string filePath)
         // the project will be updated, and it will no longer be a fallback project.
         var hostProject = new FallbackHostProject(project.FilePath, intermediateOutputPath, FallbackRazorConfiguration.Latest, rootNamespace, project.Name);
 
-        _projectSnapshotManagerAccessor.Instance.ProjectAdded(hostProject);
+        _projectManagerAccessor.Instance.ProjectAdded(hostProject);
```

Comment:
Why is this not on the dispatcher thread?

Reply:
Missed that one. Thanks!